frame/election-provider-multi-phase/src/signed.rs

```diff
@@ -395,10 +395,19 @@ impl<T: Config> Pallet<T> {
 		// Any unprocessed solution is pointless to even consider. Feasible or malicious,
 		// they didn't end up being used. Unreserve the bonds.
 		let discarded = all_submissions.len();
-		for SignedSubmission { who, deposit, .. } in all_submissions.drain() {
+		let mut refund_count = 0;
+		let max_refunds = T::SignedMaxRefunds::get();
+		for SignedSubmission { who, deposit, call_fee, .. } in all_submissions.drain() {
+			if refund_count < max_refunds {
+				// Refund fee
+				let _ = T::Currency::deposit_creating(&who, call_fee);
```

Comment:
Yeah, both are technically minting funds. Despite the name, call fee refund is minting new funds since the actual fee probably went to block producer/treasury. So I think your suggestion makes sense

Reply:
4867e2feef6dd011c24a76863b9b4c3d1dce0f23